test/benchmarks/Benchmarks.Trace/Log4netBenchmark.cs

```diff
@@ -0,0 +1,90 @@
+using System.IO;
+using BenchmarkDotNet.Attributes;
+using Datadog.Trace;
+using Datadog.Trace.Configuration;
+using Datadog.Trace.Logging;
+using Datadog.Trace.Logging.LogProviders;
+using log4net.Appender;
+using log4net.Core;
+using log4net.Layout;
+using log4net.Repository.Hierarchy;
+
+namespace Benchmarks.Trace
+{
+    [MemoryDiagnoser]
+    public class Log4netBenchmark
+    {
+        private static readonly Tracer BaselineTracer;
+        private static readonly Tracer LogInjectionTracer;
+        private static readonly log4net.ILog Logger;
+
+        static Log4netBenchmark()
+        {
+            LogProvider.SetCurrentLogProvider(new Log4NetLogProvider());
+
+            var logInjectionSettings = new TracerSettings
+            {
+                StartupDiagnosticLogEnabled = false,
+                LogsInjectionEnabled = true,
+                Environment = "env",
+                ServiceVersion = "version"
+            };
+
+            LogInjectionTracer = new Tracer(logInjectionSettings, new DummyAgentWriter(), null, null, null);
+            Tracer.Instance = LogInjectionTracer;
```

Comment:
Just wondering why we set this here? the static instance shouldn't be used should it?